src/core/graph.js

```diff
@@ -1212,3 +1212,70 @@ export function edgeToParts(
   const timestampMs = edge.timestampMs;
   return {addressParts, srcParts, dstParts, timestampMs};
 }
+
+export type GraphComparison = {|
+  +graphsAreEqual: boolean,
+  +uniqueNodesInFirst: $ReadOnlyArray<Node>,
+  +uniqueNodesInSecond: $ReadOnlyArray<Node>,
+  +nodeTuplesWithDifferences: $ReadOnlyArray<[Node, Node]>,
```

Comment:
Riffing off [this comment](https://github.com/sourcecred/sourcecred/pull/2458/files#r519002370):

What if we had a type `NodeDiff = {|+first: Node | null, +second: Node | null|}`
And then instead of having `uniqueNodesInFirst`, `uniqueNodesInSecond`, `nodeTuplesWithDifferences`, we could just have a single array of `NodeDiffs`?

Reply:
accepted, I can bake this into the other PR.